README.md

```diff
@@ -99,3 +99,61 @@ Below is a list of workloads.
 | SqlUpdates                       | Sample key-value app built on PostgreSQL with concurrent readers and writers. The app updates existing string keys |
 | SqlSecondaryIndex                | Sample key-value app built on postgresql. The app writes out unique string keys |
 | SqlSnapshotTxns                  | Sample key-value app built on postgresql. The app writes out unique string keys |
+
+## Load balancing support in SQL workloads
+
+New load balancing features are introduced in SQL workloads. The changes resulting from this new feature are visible in:
+
+* pom.xml: contains both the upstream PostgreSQL JDBC driver dependency as well as Yugabyte’s smart driver dependency.
+
+* SQL* workloads: can be started with either the Yugabyte’s smart driver or with the upstream PostgreSQL driver.
+
+* Yugabyte's smart driver: is the default driver.
+
+* Three new arguments are introduced in the SQL* workloads:
+
+   1. load_balance: It is true by default. When load_balance is `true` then YB’s smart driver is used by the sample apps. So if you have a YB cluster created with a replication factor of 3(-rf) and the total number of connections ( it is equal to the sum of reader and writer threads ) will be evenly distributed across the 3 servers. If you explicitly set load-balance to `false` then the upstream PostgreSQL JDBC driver will be used and it will be same as the current state of the sample apps.
```

Comment:
```suggestion
  * `load_balance`: It is true by default. When load_balance is `true` then YB’s smart driver is used by the sample apps. So if you have a YB cluster created with a replication factor of 3(-rf) and the total number of connections ( it is equal to the sum of reader and writer threads ) will be evenly distributed across the 3 servers. If you explicitly set load-balance to `false` then the upstream PostgreSQL JDBC driver will be used and it will be same as the current state of the sample apps.
```